src/kajol/ShopStockDetails.java

```diff
@@ -0,0 +1,86 @@
+package kajol;
+
+public class ShopStockDetails {
+	static int intialStockOfMaggie=50;
+	static int intialStockODosa=43;
+	static int intialStockOfPouches=39;
+	static int intialStockOfPanipuri=43;
+	static int intialStockOfMasala=73;
+	
+	void itemsToBePurchased(int purchasedMaggie,int purchasedDosa,int purchasedPouches,int purchasedPanipuri,int purchasedMasala) {
+		System.out.println("Items Purchased....");
+		if(purchasedMaggie>intialStockOfMaggie) 
```

Comment:
If I passpurchasedMaggie as 51, then it will goes into else part & make intialStockOfMaggie in -1, which looks wrong logically,

same for all other methods.